README.rst

```diff
@@ -165,15 +104,15 @@ report bugs, and request new features. This is the best place to post questions
 Distribution
 ------------
 
-If you want to create either source or wheel files, start by installing
-the building requirements and then executing the build module:
+If you want to create either source or wheel files, start by installing poetry and then 
+executing the build module. To install poetry we suggest using the pipx installation
```

Comment:
```suggestion
executing the build module. To install poetry we suggest using the ``pipx`` installation
```
I looked at the referenced documentation, and they tagged ``pipx`` as a literal. Whether to tag these names is random these days. We tag ``pip``, ``pipx", and ``tox`` but not ``poetry`` and other package names that are familiar words. I hate that we are are all over the place with this.